emsdk/tests/test_dyncall.py

```diff
@@ -20,19 +20,10 @@ def test_dyncall(tmpdir):
 typedef int (*type_iifid) (int, float, int, double);
 
 void foo() {
-  // Hack to force inclusion of malloc
-  volatile int x = (int) malloc(1);
-  free((void *) x);
-
   type_iifid fp = &indirect_function;
-
-  jmp_buf buf;
-  int i = setjmp(buf);
-
+  int i=5;
   printf("%d\\n", i);
   assert(fp(i, 0, 0, 0) == i);
-
-  if (i == 0) longjmp(buf, 1);
```

Comment:
Can you leave the original test in some form? This was originally included to fix a problem that is triggered specifically with setjmp/longjmp. I don't think the bug is still present, but it manifest itself in one of the last `scipy` tests, so it would be nice to be able to catch it earlier on.

Reply:
darn, I wondered why that was in there. The test works if the main.c has setjmp and longjmp, otherwise setjmp doesn't get linked in with new wasm thanks to the pretty hardcore dead-code elimination. I'll mess with the common / main.c so it works then.